setup.py

```diff
@@ -85,35 +71,24 @@ def build_extension(self, ext: Extension):
 
 setup(
     name=PACKAGE_NAME,
-    version=get_version(),
     author="Keming Yang",
     author_email="kemingy94@gmail.com",
     description="Model Serving made Efficient in the Cloud.",
     long_description=readme,
     long_description_content_type="text/markdown",
     url="https://github.com/mosecorg/mosec",
     license="Apache License 2.0",
+    use_scm_version=True,
     packages=find_packages(exclude=["examples*", "tests*"]),
     include_package_data=True,
-    classifiers=[
-        "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: Implementation :: CPython",
-        "Topic :: Software Development :: Libraries :: Python Modules",
-    ],
     python_requires=">=3.6",
```

Comment:
3.7?

Reply:
We provide the PyPI package starting from `py3.7`. `py3.6` should also work but we don't guarantee it. If we set it to `py3.7`, I guess `py3.6` users can not install from the source distribution(not sure)?